dadahee/fasting-date-counter
# add food & review test code, update minor changes in details

# Server

## feature
- test food / review 
    - detail
        - add test classes and check feature operations working normally
        - tried to consider all possible success & failure cases
    - target
        - food: api controller, repository, service
        - review: api controller, repository, service
- minor changes
    - add d-day(not fasting day count but day count from the start date of fasting) on food view
    - change the language of error messages (EN -> KO)

## linked issue
solve #10
